pages/participate.vue

```diff
@@ -80,35 +57,27 @@
             <h1>Financial Aid and Diversity Scholarships</h1>
 
                 <p>
-                    NumFOCUS and the JupyterCon team believe that diversity is the strength of our community and essential for success. To help foster this inclusive environment, we are offering scholarships and financial aid to create an event that brings together many different perspectives.  We want to include participants that represent all ethnicities, sexual orientations, religions, ages, and genders. We’d also like to introduce geo-diversity as a goal to bring attendees from around the globe since JupyterCon has been previously held in the US and now Europe.
+                    NumFOCUS and the JupyterCon team believe that diversity is the strength of our community and essential for success. To help foster this inclusive environment, we are offering scholarships for full conference passes (all talks, tutorials, social events) to create an event that brings together many different perspectives.  We want to include participants that represent all ethnicities, sexual orientations, religions, ages, genders, and geographical areas.
                 </p>
 
                 <h2>How to apply</h2>
-<p>Please use [this form](https://docs.google.com/forms/d/e/1FAIpQLScBy-7hNFDWLsejqNM0wS2qE4Q5APeXO8rlYW3kzgltq-bIZg/viewform) to apply for Diversity and Financial Aid scholarships.
-</p>                    
+
+                <p>
+                    Please use <a href="https://docs.google.com/forms/d/e/1FAIpQLScBy-7hNFDWLsejqNM0wS2qE4Q5APeXO8rlYW3kzgltq-bIZg/viewform">this form</a> to apply for Diversity and Financial Aid scholarships.
+                </p>
  
                     <h2>Dates to remember</h2>
                       <table style="width:100%">
                   <tr>
-                    <td>April 27th, 2020</td>
+                    <td>July 22</td>
```

Comment:
```suggestion
                    <td>July 15</td>
```
I remember the 15th from the meeting, or we will use 22nd?

Reply:
Originally we had the scholarships due a week after the CFP, with the idea that it would only take two weeks to review scholarship applications.